package.json

```diff
@@ -104,7 +104,7 @@
     "ws": "^8.11.0"
   },
   "engines": {
-    "node": ">=14.0"
+    "node": ">=18"
```

Comment:
are we bumping a major?

Reply:
No. Just don't want all the version test to run while testing